package.json

```
@@ -38,7 +38,7 @@
   "dependencies": {
     "@brightspace-ui/core": "^1",
     "d2l-localize-behavior": "BrightspaceUI/localize-behavior#semver:^2",
-    "@brightspace-ui-labs/multi-select": "BrightspaceUILabs/multi-select#semver:^4",
+    "@brightspace-ui-labs/multi-select": "^4.2.4",
```

Could you just make this `^4`? Typically we try to avoid including minor versions unless it specifically needs a feature in them. BSI will always take the latest version due to the `^` anyway.
```suggestion
    "@brightspace-ui-labs/multi-select": "^4",
```